client/my-sites/sidebar/sidebar.jsx

```diff
@@ -660,12 +661,22 @@ export class MySitesSidebar extends Component {
 			return null;
 		}
 
+		const adminUrl =
+			this.props.isJetpack && ! this.props.isAtomicSite && ! this.props.isVip
+				? formatUrl(
+						Object.assign( parseUrl( site.options.admin_url ), {
```

Comment:
Personally, I find seeing an rvalue as the `target` argument to `Object.assign()` a bit irritating :sweat_smile: How about using the spread operator instead?

Reply:
I don't mind too much either way. The spread might be slightly more legible, but also a bit more work. I applied this suggestion:
47ae15cd2fa7d979042b26e2258dbe1105a4aa43